scvi/utils/_differential.py

```diff
@@ -236,6 +242,18 @@ def get_bayes_factors(
                 m_permutation=m_permutation,
             )
 
+        # Adding pseudocounts to the scales
+        if pseudocounts is None:
+            logger.debug("Estimating pseudocounts offet from the data")
+            where_zero_a = densify(np.max(self.adata[idx1].X, 0)) == 0
+            where_zero_b = densify(np.max(self.adata[idx2].X, 0)) == 0
```

Comment:
this should use `get_from_registry` the "X" key because X can be in several places.